Blog-projects/Widget-window-in-Qt-Quick-app/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(WidgetWindowsInQtQuickApp VERSION 0.1 LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
This line is usually just after set(CMAKE_CXX_STANDARD 17) or some other value.
Here you're saying "the value of CMAKE_CXX_STANDARD is required" but without saying what CMAKE_CXX_STANDARD is. Either do specify it, or let Qt specify it (Qt 6 requires C++17) in which case you don't need this line, I would think?

Reply:
Removed. This should make things easier to maintain.